ouroboros-consensus-byron/test/Test/Consensus/Byron/Ledger/Golden.hs

```diff
@@ -1,5 +1,7 @@
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE RankNTypes        #-}
+{-# LANGUAGE TypeApplications  #-}
+{-# LANGUAGE DataKinds         #-}
```

Comment:
Can you sort this list? Otherwise, `stylish-haskell` will reorder it the next time we touch this file.

Reply:
:disappointed: